corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -30,13 +30,14 @@ hqDefine("data_dictionary/js/data_dictionary", [
 
         self.init = function (groupDict, changeSaveButton) {
             _.each(groupDict, function (properties, group) {
-                var groupObj = propertyListItem(group, true, group, self.name);
+                var groupObj = propertyListItem(group, '', true, group, self.name);
```

Comment:
how come this one is empty string?

Reply:
this is a property group so label field is not displayed here, hence the empty string.